src/core/ext/transport/chttp2/transport/OWNERS

```diff
@@ -0,0 +1,2 @@
+set noparent
```

Comment:
Is this actually useful, considering that OWNERS doesn't actually require approval, it just causes you to be CCed on PRs?